client/packages/common/src/hooks/useUrlQuery/useHandleUrlQueryParams.ts

```diff
@@ -0,0 +1,65 @@
+import { useUrlQuery } from './useUrlQuery';
+import { Column } from '@openmsupply-client/common';
+import { FilterController } from '../useQueryParams';
+
+// This hook uses the state of the url query parameters to provide query parameters and update methods to tables.
+
+export const useHandleUrlQueryParams = (filterIndex: string) => {
+  const { urlQuery, updateQuery } = useUrlQuery();
+
+  const updateSortQuery = (column: Column<any>) => {
+    const currentSort = urlQuery?.['sort'];
+    const sort = column.key as string;
+    if (sort !== currentSort) {
+      updateQuery({ sort, dir: '', page: '' });
```

Comment:
Only show `dir` when descending (assume "ascending" is default) as the url gets necessarily cluttered IMHO.

Also, seems appropriate to reset to Page 1 again when changing the sort order.